src/main/java/com/ericsson/ei/frontend/exceptions/EiBackendInstancesException.java

```diff
@@ -0,0 +1,35 @@
+/*
+   Copyright 2017 Ericsson AB.
```

Comment:
Should an new file have 2017 in the copyright?

Reply:
done